Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py

```diff
@@ -32,6 +32,53 @@ def plot_hist(self, n_cols=3):
             self.data.state.sel(circuit_depth=circuit_depth).plot.hist(ax=ax, xticks=range(4))
         plt.tight_layout()
 
-    def plot_fidelity(self):
+    def plot_decay(self):
```

Comment:
I would add a small docstring describing what it returns if this is a public method accessible to the user

Reply:
Hey Arthur, thanks for the suggestion.

I added docstrings everywhere and also renamed the methods to make them more clear. Please tell me what you think now.